### ISSUE TITLE
Getting started fails at `elm-install`

### ISSUE THREAD

**Issue by manuscrypt** [opsb/elm-native-expo-counter#1]
$ elm-install -t
Resolving packages...

  The source of package ohanhi/elm-native-ui is set to git@github.com:opsb/elm-native-ui which would
  be install to elm-stuff/opsb/elm-native-ui. This would cause a conflict
  when trying to compile anything.

  The name of a package must match the source url's path.

  ohanhi/elm-native-ui <=> opsb/elm-native-ui

**Comment by manuscrypt** [comment]
Solved by removing `~/.elm/0.18.0/package/ohanhi` that was there from previous experiments.

**Comment by manuscrypt** [comment]
Actually, that didn't help... :(

**Comment by opsb** [comment]
Are you running the latest version of elm-github-install? The dependency resolver has been rewritten lately so if you haven't got the latest version the behaviour might be different. I'll try a clean checkout tomorrow myself and see if it works as is for me.

**Comment by opsb** [comment]
I tried a fresh checkout and it seems to work fine with elm-github-install 1.0.1, @manuscrypt can you confirm which version of elm-github-install you're using?

**Comment by opsb** [comment]
ohahni has merged my [PR](https://github.com/ohanhi/elm-native-ui/pull/84) so I've updated the dependency to point to the upstream repo. Let me know if that works for you.

**Comment by manuscrypt** [comment]
Unfortunately, i totally bombed my npm when trying to "fix" this, so I cannot tell you right now (or ever) what version I had of elm-github-install. Once I have fixed my setup, I'll try what you suggested. Thank you for your support!

**Comment by manuscrypt** [comment]
Thank you very much for your support and patience in elm slack. With your help, I got past this step. To summarize, what i did:

- clean node installation by installing nvm by script: https://github.com/creationix/nvm#install-script
- then run `nvm 7.2.1`
- run `npm install -g elm-github-install` (note, no sudo, I had /usr/lib/node_module/folders with root user permissions, so I deleted those as well)
- and finally `elm-install` (currently at version 1.0.1)

Up to the next step!
